tests/unit/synthesizer/test_ctgan.py

```diff
@@ -284,7 +284,6 @@ def test__validate_discrete_columns(self):
 
         Note:
             - could create another function for numpy array
-            - TODO: it is currently a integration test, needs to be changed to a proper unit test
```

Comment:
Do we have an issue to improve testing for `CTGAN`?

Reply:
I created one to fix this specific test (I linked it in the description). We don't have an issue to fix all tests though.